.github/actions/python-environment/action.yml

```diff
@@ -18,22 +18,44 @@ inputs:
     required: true
     default: "."
 
+  extras:
```

Comment:
Please switch 'Setup python and poetry environment' to 'Set up python and poetry environment'